EIPS/eip-X-ASIC-resistant-block-mining.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: <to be assigned>
+title: Poisoning the Well
+author: David Stanfill <david@airsquirrels.com> 
+discussions-to: <email address>
+status: Draft
+type: Standards Track
+category Core
+created: 2018-04-03
+---
+
+
+## Simple Summary
+
+This EIP attempts to break ASIC miners specialized for the current ethash
+mining algorithm.
+
+
+## Abstract
+
+There are companies who currently have dedicated hardware based ethereum miners in
+production, and probabalistically actively mining.  This EIP aims to "Poison
```

Comment:
Should read: "and probably actively mining"

I also recommend rewording to say, "and may be actively mining with that hardware".  EIPs are not the place for speculation, and the only "facts" we have are that there is one company actively selling pre-orders for an EthHash ASIC.